Samples/word-add-in-get-set-edit-openxml/README.md

```diff
@@ -0,0 +1,113 @@
+---
+page_type: sample
+urlFragment: word-get-set-edit-ooxml
+products:
+  - office-add-ins
+  - office-word
+  - office
+  - office-365
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: '01/24/2021 10:00:00 AM'
+description: 'Learn how to get, edit, and set OOXML content in a Word document.'
+---
+
+# Get, edit, and set OOXML content in a Word document with a Word add-in
+
+## Summary
+
+Shows how to get, edit, and set OOXML content in a Word document. This sample Word add-in provides a scratch pad to get Office Open XML for your own content and test your own edited Office Open XML snippets.
+
+## Features
+
+- Shows the **getSelectedDataAsync** and **setSelectedDataAsync** methods to get and set OOXML content in Word 2013.
+- Shows the **getOoxml** and **setOoxml** methods to get and set OOXML content in Word 2016 and later.
+
+## Applies to
+
+- Word on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+- Visual Studio 2019 or later with the **Office/SharePoint development** workload installed.
+- Microsoft 365 - You can get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 developer subscription.
+
+## Run the sample
+
+1. Open the **WD_OpenXML_js.sln** file in Visual Studio 2019 or later.
+2. Choose **Debug > Start Debugging**, or choose the F5 key.
+3. On the **Home** tab, in the **OOXML Editor** group, choose **Open**.
+
+The SampleDoc.docx file will open in Word since it is set as the **StartAction** property of the add-in. The document contains a range of common rich content types for Word, including the following:
```

Comment:
I think in this case, we keep the colon after following
```suggestion
The SampleDoc.docx file will open in Word since it's set as the **StartAction** property of the add-in. The document contains a range of common rich content types for Word, including the following:
```